recipes/glew/all/conanfile.py

```diff
@@ -17,10 +17,12 @@ class GlewConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "with_egl": [True, False]
```

Comment:
Other then that I would have named the option `enable_egl` or just `egl`, LGTM>

Reply:
What's our convention on this? It's not in our docs AFAIK

Reply:
There is none.

Perhaps this option naming is fine, but maybe we should standardize certain common options.
e.g. for compression, use always `with_zlib` instead of `enable_zlib/enable_compression`. Or for libjpeg: use `with_libjpeg`, ...

Reply:
I tend to use `with_` prefix when option refers to a dependency.